src/pages/PageCamera.vue

```diff
@@ -43,6 +56,36 @@ const post = reactive({
   photo: null,
   date: Date.now()
 })
+
+const imageCaptured = reactive ({
+  captured: false
+})
```

Comment:
const imageCaptured = reactive ({
  captured: false
})

in such cases (boolean, strings, non-objects) you can use ref (for objects you also can use refs)

const imageCaptured: Ref<boolean> = ref(false)                   TS
const imageCaptured = ref(false)                                            JS

....
and then use like this:


 context.drawImage(video.value, 0, 0, canvas.value.width, canvas.value.height)
 imageCaptured.value= true